types.go

```diff
@@ -127,20 +127,14 @@ const (
 
 // Navigation Status (NMEA 4.1 and later)
 const (
-	// NavStatusAutonomous is Autonomous mode
-	NavStatusAutonomous = "A"
-	// NavStatusDifferential is Differential Mode
-	NavStatusDifferential = "D"
-	// NavStatusEstimated is Estimated (dead-reckoning) mode
-	NavStatusEstimated = "E"
-	// NavStatusManualInput is Manual Input Mode
-	NavStatusManualInput = "M"
-	// NavStatusSimulated is Simulated Mode
-	NavStatusSimulated = "S"
-	// NavStatusDataNotValid is Data Not Valid
-	NavStatusDataNotValid = "N"
-	// NavStatusDataValid is valid
-	NavStatusDataValid = "V"
```

Comment:
I propose deleting all but these 2 and mark these deprecated

```go
	// NavStatusSimulated is Simulated Mode
	// Deprecated: correct is NavStatusSafe
	NavStatusSimulated = "S"
	// NavStatusDataValid is valid
	// Deprecated: correct is NavStatusNotValid
	NavStatusDataValid = "V"
```

in that case when someone depend most used values - we do not break their apps immediately but when you depend incorrect enums - we force you to fix your code (replace deleted values)

Reply:
That makes sense

Reply:
The deprecation message should be 

```
Deprecated: use NavStatusNotValid
```